charts/argo-cd/README.md.gotmpl

```diff
@@ -28,7 +28,7 @@ redis-ha:
   enabled: true
 
 controller:
-  enableStatefulSet: true
+  replicas: 2
```

Comment:
2 replica for the application controller is a bad default. The controller cannot shard its work if Argo CD manages only one cluster.

The shard number of a managed cluster is calculated via hostname:
https://github.com/argoproj/argo-cd/blob/v2.4.0/cmd/util/cluster.go#L133

Reply:
I guess number of replicas comes from this example: https://argo-cd.readthedocs.io/en/stable/operator-manual/high_availability/#argocd-application-controller. Do you want to use 3 for HA setups?